Add blog post: "Where Does the Value Live?" (#34)
mattfelten/mattfelten.com: src/content/posts/2025-02-12-design-process.md

---
title: Where Does the Value Live?
date: 2025-02-12
---

I learned the design process the way most of us did. Research, define, ideate, prototype, test, iterate. It's a good framework. It taught me to slow down, to ask questions before jumping to solutions, to listen to people who aren't me.

> I was mostly self-taught. No mentor. No team of senior designers showing me the ropes. Process gave me something to hold onto when I wasn't sure I was doing any of this right. It's how I learned rigor. It's how I learned empathy. I owe a lot to it.

Author: I wasn't mostly self-taught, but I didn't work on big teams. A lot of my knowledge came from doing a lot of work. Talking with and learning from other designers. Articles on the internet. Trying things out and keeping what was a good use of time.

> But somewhere along the way, I think the industry started treating process like the destination instead of the road. We got really good at talking about _how_ we work &mdash; the diamond diagrams, the workshops, the ceremony &mdash; and a little less focused on the outcomes themselves.

Author: > I think the industry started treating process like the destination instead of the road. 

I agree with this but how can we lighten it. I don't want to blanketly say the whole industry is bad or anything.

> Frank Chimero has this idea that a designer's superpower is bridging the present into the future. Taking the exciting possibility and making it real. That takes intuition. A point of view. Some guts. Those things don't live in a process diagram.

Author: Maybe something like "[Frank Chimero](https://frankchimero.com/) wrote in his book [The Shape of Design](https://shapeofdesignbook.com/) that..."

After that first sentence about the paraphrased quote make sure it's clear that the rest of it is my own addition and not part of the quote.

I don't think process is the problem. I think mistaking process for the work is.

---

And then execution got faster.

> Not just AI &mdash; better design tools, component libraries, mature frontend frameworks, tighter feedback loops across the board. The whole stack of making things got cheaper. AI is part of that, the same way Figma was part of it, the same way Sketch was before that. Tools evolve. The interesting question isn't about which tool. It's what changes when the cost of trying an idea drops to nearly zero.

Author: ```suggestion
We got better design tools, component libraries, mature frontend frameworks, tighter feedback loops across the board. AI. The whole stack of making things got cheaper. AI is part of that, the same way Figma was part of it, the same way Sketch was before that. Tools evolve. The interesting question isn't about which tool. It's what changes when the cost of trying an idea drops to nearly zero.
```

> Here's what changed for me: I stopped trying to pick the "right" idea upfront and started trying more ideas, faster. The process flipped from _narrow down, then build_ to _build many, then narrow down._

Author: > I stopped trying to pick the "right" idea upfront and ....

I don't want it to sound like I was just spending a bunch of time deciding on a single idea to pursue. A lot of projects take many iterations heading in multiple different directions. The main thing though is that this all got faster.

I had a project recently &mdash; a third-party integration, fast-paced partnership, needed to ship as soon as possible. I created seven working React prototypes in two days. Not mockups. Functional code, running in the app. They covered a full spectrum: one redesigned the entire customer dashboard, one was the smallest change possible, and the rest fell in between. I clicked through each of them, felt which ones were getting closer, and funneled down to a shortlist.

> Then a half-day of craft &mdash; bouncing between Figma and code. Larger layouts and workflows were faster to explore in React. Specific UI elements, like a card component, were faster to iterate in Figma. I went through about seven iterations of that card before it felt right. Then brought the design into code.

Author: > Larger layouts and workflows were faster to explore in React. Specific UI elements, like a card component, were faster to iterate in Figma.

This is seeming to be true in general, not just in this project. The larger the feature, the more steps or screens to make, the more edge cases, prototyping in code is way faster.

Concept to production. Under a week.

---

> When you can prototype that fast, the bottleneck moves. It's not execution anymore. It's decision-making. And no process can make that decision for you.

Author: > And now process can make that decision for you.

I don't like this. That's exactly what an overly-controlling process tries to do, take all the decision-making out of it. All the subjectivity, all the opportunity for magic to happen.

I've had to get better at evaluating, not just creating. Looking at seven prototypes side by side and knowing which one to ship &mdash; that requires taste. Designers love that word but rarely define it. For me it's pretty simple:

- Does this feel right for the person using it?
- Does it solve the real problem, not just the stated one?
- Is it as simple as it can be?

---

> The spectrum approach is deliberate. I prototype across a range, from smallest-viable-change to full-redesign, because of the boiling frog problem. Small incremental additions slowly degrade a UI without anyone noticing. One more button here. One more settings panel there. Eventually the whole thing is a mess and nobody can point to when it happened.

Author: > The spectrum approach is deliberate. I prototype across a range, from smallest-viable-change to full-redesign, because of the boiling frog problem.

I don't like how this reads. The point is fine though. Let's try a new way of saying it.

Sometimes the right answer really is the smallest change. Sometimes you realize the whole thing needs rethinking. You don't know until you see both ends side by side.

---

> None of this means process is useless. Research still matters. Understanding users still matters. But the ratio changed for me. I spend less time in long discovery phases and more time in rapid build-evaluate-decide loops. The thinking happens _through_ making, not before it.

Author: ```suggestion
Don't get me wrong, I'm not anti-process. Research still matters. Understanding users still matters. But the ratio changed for me. I spend less time in long discovery phases and more time in rapid build-evaluate-decide loops. The thinking happens _through_ making, not before it.
```

Author: The tools we have allow us to use "process" as tools in our toolbelt. Depending on the problem we're solving, we can pull out just the tools we need to get to a great outcome, and I think we're able to do that faster than ever now.

> I've seen thoughtful process produce great work. I've also seen process become a safety net &mdash; follow the steps, and the outcome feels secondary. I've been guilty of both. The point isn't to skip research or move fast and break things. The point is that process should serve your judgment, not replace it.

Author: ```suggestion
I've seen thoughtful process produce great work. I've also seen process become a safety net &mdash; follow the steps, and the outcome feels secondary. I've been guilty of that. The point isn't to skip research or move fast and break things. The point is that process should serve your judgment, not replace it.
```

---

> Here's the other thing I've been wrestling with. Design work is bigger than most of us define it.

Author: I like to say everyone takes part in good user experience, not just designers. How we prioritize projects is design work. How a feature is implemented in code is design work. It's all design because it all has an impact on what the user sees at the end, how it feels.

I scope projects. I define API shapes with backend engineers. I refine tasks for the engineering team. I work with product on roadmaps &mdash; not just _what_ are we building but _why_ are we building it.

When I see a UX papercut, half the time I file a ticket. The other half, I just fix it in code. When the team is under pressure to ship, I pick up engineering work.

> This isn't about being a unicorn. It's about recognizing that design decisions happen everywhere &mdash; in the API contract, in the scope conversation, in what you decide _not_ to build. If I limit myself to Figma, I'm missing half the places where the user experience actually gets shaped.

Author: ```suggestion
This isn't about being a unicorn. It's about recognizing that design decisions happen everywhere &mdash; in the API contract, in the scoping conversation, in what you decide _not_ to build. If I limit myself to Figma, I'm missing half the places where the user experience actually gets shaped.
```

---

> Not every team works this way. I've been on teams with hard walls between roles, where being a designer felt like being a tollbooth. But when a team values someone working across those lines, the leverage is enormous. That's the bridge Chimero is talking about &mdash; between the present and the future. It gets built in the space between disciplines.

Author: I don't like tollboth here. Maybe something like "where being a designer felt like a obligatory-and-time-consuming step in the product development cycle."

We also don't need to bring Frank Chimero up again here. I understand that it ties it all together, but I think it's a little too loose of a connection.

---

My process isn't a diamond diagram. It's a set of values.

- Stay close to the real materials &mdash; code, users, data.
- Try more things than you think you need to.

> - Have the guts to make a call.

Author: "Have the guts to make a call" is a strong statement. I like it but I think it could be better. Can I see some other options for it?
